vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/utils/CBORDataItemUtils.kt

```diff
@@ -0,0 +1,33 @@
+package io.mosip.vercred.vcverifier.utils
+
+import co.nstant.`in`.cbor.CborDecoder
+import co.nstant.`in`.cbor.CborEncoder
+import co.nstant.`in`.cbor.CborException
+import co.nstant.`in`.cbor.model.Array
+import co.nstant.`in`.cbor.model.DataItem
+import co.nstant.`in`.cbor.model.MajorType
+import co.nstant.`in`.cbor.model.Map
+import co.nstant.`in`.cbor.model.UnicodeString
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+
+object CborDataItemUtils {
```

Comment:
This should be reused from pixelpass library

Reply:
SignatureVerifier's verify method expects signData to be in ByteArray but we will be having the signData as DataItem, to convert the signData to byteArray and vice versa we are using this util class